HW1/P2/P2a.py

```diff
@@ -1,3 +1,34 @@
+import matplotlib.pyplot as plt
 from P2 import *
+import pyspark
 
 # Your code here
+TOTAL_PIXELS = 2000
+
+def calcBase(i):
+    return i / 500.0 - 2
+
+def calcX(ij):
+    return calcBase(ij[1])
+
+def calcY(ij):
+    return calcBase(ij[0])
+
+if __name__ == '__main__':
+    imageIJ = []
+    for i in xrange(TOTAL_PIXELS):
+        for j in xrange(TOTAL_PIXELS):
+            imageIJ.append((i, j))
+
```

Comment:
Was creating the computational grid in this way a bottleneck in terms of computation time? I feel like it could become an issue because you're doing nested 'for' loops. 

I did it like this, using "map":
q = range(0,4000)
X = sc.parallelize(q,100)
coordinates = X.flatMap(lambda x: [(x,y) for y in q])